wolfssl/openssl/evp.h

```diff
@@ -102,7 +102,7 @@ typedef union {
 
 typedef struct WOLFSSL_EVP_MD_CTX {
     unsigned char macType;
-    WOLFSSL_Hasher hash;
+    ALIGN16 WOLFSSL_Hasher hash;
```

Comment:
Why the new need for 16 byte alignment here?  macType should be moved below hash if this alignment is needed to avoid wasting 15 bytes.